### ISSUE TITLE
Tests unitarios lógica de partida

### ISSUE THREAD

**Issue by Juanguppy** [UNIZAR-30226-2024-02/Back-end#74]

**Comment by 737275** [comment]
¿Han sido probadas las funciones? Estaba lanzando peticiones a la ruta de `realizarManiobra` y su función correspondiente me están dando errores que parecen que este código nunca se ha ejecutado, por ejemplo la función `comprobarTerritorio` se está llamando con un número incorrecto de parámetros y la función `numJugador` no está esperando la resolución de la promesa y está generando fallos debido a no tener los valores a tiempo durante la ejecución de las comprobaciones...

Si este es el caso, se ha de reabrir el issue correspondiente a la creación de la lógica de la partida.

**Comment by Juanguppy** [comment]
> ¿Han sido probadas las funciones? Estaba lanzando peticiones a la ruta de `realizarManiobra` y su función correspondiente me está dando errores que hacen creer que este código nunca se ha ejecutado, por ejemplo la función `comprobarTerritorio` se está llamando con un número incorrecto de parámetros y la función `numJugador` no está esperando la resolución de la promesa y está generando fallos debido a no tener los valores a tiempo durante la ejecución de las comprobaciones...
> 
> Si este es el caso, se ha de reabrir el issue correspondiente a la creación de la lógica de la partida.

El issue de lógica de partida nunca se ha cerrado, porque no está terminada ( #25 ). 
Respecto a lo que hay, por lo que he debuggeado parece que está literalmente hecho con chatGPT 3.5 en español y que ni se ha probado, lo cual es una práctica que no me parece correcta como encargado de calidad del proyecto. Si se usa chatGPT que se use como auxiliar y si se utiliza para hacer el esqueleto de algo que al menos se pruebe y se debuggee. Y de todas formas está copilot gratis con el plan de estudiante que funciona bastante mejor que gpt 3.5 el cual últimamente se fuma buenos porros.

**Comment by FabiMur** [comment]
No, no esta hecho con ChatGPT. Un poco de cuaidado antes de ponerse a acusar a la gente. El codigo ese lo escribi simplemente como primer boceto, fui parcheando cosas sobre la marcha y no he podido ponerme a probarlo, por eso hay errores del estilo.
